### ISSUE TITLE
Issue with plugins

### ISSUE THREAD

**Issue by dhanugupta** [cgswong/docker-elasticsearch#2]
Hi

Docker image builds and run perfectly fine. but all the plugins are not working.
when you try to reference /_plugin/head or bigdesk nothing showing up on the browser.

is this expected?

**Comment by cgswong** [comment]
No. Thanks for letting me know. I'm looking into this and will also try and add some tests in my build process to catch this in future.
